pac4j-http/src/main/java/org/pac4j/http/credentials/authenticator/test/SimpleTestUsernamePasswordAuthenticator.java

```diff
@@ -48,6 +50,10 @@ public void validate(final UsernamePasswordCredentials credentials) {
         if (CommonHelper.areNotEquals(username, password)) {
             throwsException("Username : '" + username + "' does not match password");
         }
+        final HttpProfile profile = new HttpProfile();
```

Comment:
As the `AuthenticatorProfileCreator` is now by default, create the user profile during authentication.